chef_master/source/perform_compliance_scan.rst

```diff
@@ -131,46 +69,91 @@ Before you can see if your nodes are compliant, you need to have the profiles yo
+Using the Audit Cookbook
+------------------------------------------------------
 
-Once you have the audit cookbook installed on your Chef server and your profiles are ready to use in Chef Automate, you must add the ``audit::default`` recipe to your nodes' run-lists. The audit cookbook can be reused for all nodes because all node-specific configuration is done through Chef attributes.
+The recommended method for using the audit cookbook is via a wrapper cookbook. A wrapper cookbook is a regular cookbook that includes recipes from other cookbooks. In case the wrapper cookbook will use ``include audit::default``.
 
-As part of running the audit cookbook, the InSpec gem is installed onto your nodes to run the profile(s) you specify.
+Create the Wrapper Cookbook
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-The ``default`` recipe requires a ``node['audit']['profiles']`` attribute to be set. For example, it can be defined as part of a JSON-based role or environment file:
+Generate the wrapper cookbook:
 
-.. code-block:: json
+.. code-block:: ruby
 
-   {
-     "audit": {
-       "profiles": [
-         {
-           "name": "DevSec Linux Security Baseline",
-           "compliance": "<username>/linux-baseline"
-         },
-         {
-           "name": "DevSec SSH Baseline",
-           "compliance": "<username>/ssh-baseline"
-         }
-       ],
-       "reporter": "chef-server-automate"
-     }
-   }
+  chef generate cookbook mycompany_audit
 
-.. note:: The ``audit`` cookbook also requires a time synchronization policy to be in place, such as Network Time Protocol (NTP).
+Place the following in mycompany_audit/metadata.rb:
 
-You can add the ``audit::default`` recipe to an existing run-list; however, in the example below, we will bootstrap a node with the ``audit::default`` recipe and the JSON shown above to do a series of baseline checks against a new node.
+.. code-block:: ruby
+
+  depends 'audit'
+
+Add the following in mycompany_audit/recipes/default.rb:
+
+.. code-block:: ruby
+
+  include_recipe 'audit::default'
+
+Generate the default attributes file:
+
+.. code-block:: ruby
+
+  chef generate attribute default
+
+Configure the ``audit`` cookbook reporter to send scan data to Automate in the mycompany_wrapper/attributes/default.rb file. ``audit`` cookbook supports many different configurations, see :doc:`Supported Configurations </audit_supported_configurations>`.
+
+.. code-block:: ruby
+
+  default['audit']['reporter'] = 'chef-server-automate'
+
+To run the baseline compliance profile for Windows or Linux, depending on which OS runs on the node,
+add the following code in mycompany_audit/attributes/default.rb:
+
+.. code-block:: ruby
+
+  default['audit']['reporter'] = 'chef-server-automate'
+  case node['os']
+  when 'linux'
+    default['audit']['profiles'] = [
+     {
+        'name': 'DevSec Linux Security Baseline',
+        'compliance': 'admin/linux-baseline'
```

Comment:
In order to use `admin/` they will need to make sure that these profiles are downloaded as the `admin` user. See Step 1:
http://blog.jerryaldrichiii.com/chef_automate/2017/08/04/compliance-using-the-audit-cookbook.html#download-the-necessary-profiles

Reply:
Thanks for pointing that out-

Reply:
Done.